src/web_app/css/main.css

```diff
@@ -104,6 +104,22 @@ footer > div.active {
   opacity: .8;
 }
 
+div#privacy {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  /* Makes sure to stay behind room links so they can be clicked */
+  z-index: -1
```

Comment:
Fair enough :). 

As is should be OK except I'm wondering what the margins are for. I don't think you'll need these if you set the absolute position with something like `bottom: 30px; right: 20px;`.